### ISSUE TITLE
Test coverage for slas commands

### ISSUE THREAD

**Issue by tobiaslohr** [SalesforceCommerceCloud/sfcc-ci#288]
#270 has introduced as breaking change as we've upgraded to a new major version of node-fetch. This caused slas commands to fail, reported in #280. A fix has been provided as part of #285. However, we should add unit tests and CLI functional tests to have some safeguards in place.

**Comment by tobiaslohr** [comment]
@hnestmann @johnboxall @jbachelet Any thoughts from your side?

**Comment by hnestmann** [comment]
Yes I am fully on board to increase test coverage. Howeber - would unit tests help if dependencies change? Maybe we should think about integration tests with github actions or the like

**Comment by tobiaslohr** [comment]
True for this particular issue. Having a CLI based test would have caught
it however.

**Comment by hnestmann** [comment]
I will add unit tests for sure though.
